adapters/build.gradle

```diff
@@ -24,7 +24,7 @@ android {
 }
 
 dependencies {
-    compileOnly "com.android.support:recyclerview-v7:${supportLibraryVersion}"
+    compileOnly "androidx.recyclerview:recyclerview:${androidXVersion}"
```

Comment:
This means we need to upgrade all projects (samples) using the adapter to AndroidX as well?

Reply:
Yes, when we (eventually) upgrade the examples, but I think we can just do that on a case-by-case basis as needed since there otherwise isn't any functional changes.